### PR TITLE
Rebalances flashes

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
:cl: Cobby
experiment: Flashes have been rebalanced to be more powerful
/:cl:

### CODE REVIEW

**Comment by SweaterKittens** [comment]
YOU FUCKER I WAS GONNA DO THIS

<sub>but actually thanks for doing this I've been so busy</sub>

**Comment by GunHog** [comment]
Make Flashes Great Again! (And not a complete parapen to borgs)

**Comment by ShadowDeath6** [comment]
Might be a shittier way of doing it but you could make it so borgs can't be flashed again while they're weakened

**Comment by kevinz000** [comment]
It already does a soft stun 
The problem is how fast mob subsystem ticks as sometimes it goes away immediately.

**Comment by ExcessiveUseOfCobblestone** [comment]
this puts them on their back

**Comment by jughu** [comment]
H Y P E S P E E D M E R G E

**Comment by ExcessiveUseOfCobblestone** [comment]
it now does the same amount to a person that it does a borg hmmmm....

**Comment by XDTM** [comment]
You could just keep stun there and increase it to 2

**Comment by ExcessiveUseOfCobblestone** [comment]
BUT YOU WON'T FEAR THE FLASH IF YOU DONT SEE PEOPLE DROPPIN

**Comment by Cyberboss** [comment]
Stun is shit compared to weaken

File: code/modules/assembly/flash.dm [hunk 1]
```diff
@@ -114,7 +114,7 @@
 		var/mob/living/silicon/robot/R = M
 		add_logs(user, R, "flashed", src)
 		update_icon(1)
-		M.Weaken(6)
+		M.Weaken(2)
```
Comment (PKPenguin321) [inline code/modules/assembly/flash.dm]: I don't think it was this low even pre nerf

Comment (ExcessiveUseOfCobblestone) [inline code/modules/assembly/flash.dm]: probably but the only people that don't have alternative stuns as apart of their arsenal is clockcult now [ok and gangs] which has a variety of ways to stun [flash > judi glasses > flash > get forever stun].

If you check to see what it was previously hmu so I can compare between this one and that one and make a "logical conclusion".

Comment (PKPenguin321) [inline code/modules/assembly/flash.dm]: Code diving shows it used to be rand(5,10)

Comment (GunHog) [inline code/modules/assembly/flash.dm]: I believe 3 was the old value (Possibly set by Yackemflam). While I do not feel that flashes should stun for as long as they do, they should stun long enough for someone to be able to quickly emag a borg or yank its cell.

Comment (PKPenguin321) [inline code/modules/assembly/flash.dm]: https://github.com/tgstation/tgstation/pull/5092/files

Red line 105

**Comment by dannno** [comment]
i think its fine as it is tbh

**Comment by XDTM** [comment]
You guys realize this is a science buff right? Not that i mind.

**Comment by Durkel** [comment]
>borg buff

Get out silicon lovers.

**Comment by Kromgar** [comment]
Remove telescopic baton when?

@XDTM 

You realize synthetic flashers were a thing before they nerfed flashes to the ground right? They were single use flashes you would put in to make borgs

**Comment by Anonmare** [comment]
Today. I will remind them.

I will teach them to fear the Flash.

File: code/modules/assembly/flash.dm (hunk 1 above)
Comment (GunHog) [inline code/modules/assembly/flash.dm]: That is pretty extreme. That would mean it was even **worse** than the current flash. Note, did you check that Weaken() did the same thing to silicons back then as it does now?

**Comment by FantasticFwoosh** [comment]
**RAEF EHT ESREVER HSALF**

> Recovers you from regular stuns at expense of eye damage.

**Comment by ExcessiveUseOfCobblestone** [comment]
@PKPenguin321 I made it random between 2 [my value] and 6 [current value], so you will fear the flash and the almighty RNGesus

**Comment by PKPenguin321** [comment]
That's actually making flashes less scary than before though, not sure what the goal here is

**Comment by ExcessiveUseOfCobblestone** [comment]
it's a social experiment XD

**Comment by Kromgar** [comment]
YOU CANT LOCK UP THE DARKNESS

File: code/modules/assembly/flash.dm [hunk 2]
```diff
@@ -90,7 +90,7 @@
 		if(M.flash_act(1, 1))
 			M.confused += power
 			terrible_conversion_proc(M, user)
-			M.Stun(1)
+			M.Weaken(2)
```
Comment (GunHog) [inline code/modules/assembly/flash.dm]: Why not bump this up to `Weaken(3)`?

Comment (ExcessiveUseOfCobblestone) [inline code/modules/assembly/flash.dm]: If you make it stun too long then revheads and revs who are recently flashed are fugged.

Comment (GunHog) [inline code/modules/assembly/flash.dm]: But you can just shake them up!

Comment (PKPenguin321) [inline code/modules/assembly/flash.dm]: a weaken 2 is weak as fuck, barely a disarm push

flashes used to be a weaken(5)

Comment (ExcessiveUseOfCobblestone) [inline code/modules/assembly/flash.dm]: TIME TO REALLY FEAR THE FLASH

**Comment by ExcessiveUseOfCobblestone** [comment]
HAVE I GONE MAD

**Comment by PKPenguin321** [comment]
i'm cool with this

**Comment by RandomMarine** [comment]
FLASH

AH AH

SAVIOR OF THE UNIVERSE

**Comment by Arianya** [comment]
Thank god someone managed to buff flashes without getting in a weird rutt about telebatons

**Comment by BeeSting12** [comment]
finally a reason to carry the flash

**Comment by davidchan13** [comment]
Carbons getting fucked over as hard by a stun as silicons? Oh this is gonna end well! :+1:

**Comment by GunHog** [comment]
It is only fair, tbh.

**Comment by ExcessiveUseOfCobblestone** [comment]
fuck you shitty admins

**Comment by sawrge** [comment]
keep flashes current effects on humans and then get rid of telebatons anyway tbh

**Comment by Arianya** [comment]
i will end you sawrge

**Pull request merged**